client/components/tinymce/plugins/wpcom-track-paste/plugin.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import tinymce from 'tinymce/tinymce';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+const debug = debugFactory( 'calypso:tinymce-plugins:wpcom-paste-track' );
+
+const SOURCE_GOOGLE_DOCS = 'google_docs';
+const SOURCE_UNKNOWN = 'unknown';
+
+function trackPaste( editor ) {
+	debug( 'init' );
+
+	const store = editor.getParam( 'redux_store' );
+	if ( ! store ) {
+		debug( 'store not found, do not bind events' );
+		return;
+	}
+
+	/**
+	 * Records the event.
```

Comment:
"the event"

maybe something like _"Records paste events in the editor"_ could be a little more helpful when people find this function out of context.